thehive/test/resources/data/CaseTag.json

```diff
@@ -4,5 +4,7 @@
   {"from": "case2", "to": "tagt1"},
   {"from": "case2", "to": "tagt2"},
   {"from": "case1", "to": "tagt1"},
-  {"from": "case1", "to": "tagt3"}
+  {"from": "case1", "to": "tagt3"},
```

Comment:
In the branch "develop-th4" CaseTag.json is not used for freetags. Freetags are directly included in Case.json